Yurkantovich/BulkThumbnailCreator/BulkThumbnailCreator/ManagmentApp/ManagmentApp.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace BulkThumbnailCreator
+{
+    public class ManagmentApp
+    {
+        public async Task RunAplicationAsync()
+        {
+            CancellationTokenSource tokenSource = new CancellationTokenSource();
+            CancellationToken token = tokenSource.Token;
+
+            while (!token.IsCancellationRequested)
```

Comment:
not correct usage of CancellationToken - 'While' should not have dependency on CancellationToken (or at least not only on it, but on finishing operation itself)